inc/templates/edit-form.tpl.php

```diff
@@ -18,7 +18,7 @@
 <script type="text/html" id="tmpl-shortcode-ui-field-text">
 	<div class="field-block">
 		<label for="{{ data.attr }}">{{ data.label }}</label>
-		<input type="text" name="{{ data.attr }}" id="{{ data.attr }}" value="{{ data.value }}" placeholder="{{ data.placeholder }}"/>
+		<input type="text" name="{{ data.attr }}" id="{{ data.attr }}" value="{{ data.value }}" placeholder="{{ data.placeholder }}" {{{ data.custom }}}/>
```

Comment:
Can we call this `data.attributes`? `custom` isn't very clear, imo.

Also, `placeholder` should be migrated to this new pattern, and we should have some backwards compat shim.